### PR TITLE
Fix peerDep of @prisma/client to match dependency

### PR DESCRIPTION
**Pull request by martin31821**
## Information

| Type                  | Breaking change |
| --------------------- | --------------- |
| Chore | No          |

### CODE REVIEW

File: packages/orm/prisma/package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
     "prisma": "^4.0.0"
   },
   "peerDependencies": {
-    "@prisma/client": "4.0.0",
+    "@prisma/client": "^4.0.0",
```
Comment (Romakita) [inline packages/orm/prisma/package.json]: It should be >=4.0.0

**Review by Romakita — CHANGES_REQUESTED** [review]

File: packages/orm/prisma/package.json (hunk 1 above)
Comment (martin31821) [reply packages/orm/prisma/package.json]: Should I change all deps to >=4.0.0 then?

Comment (Romakita) [reply packages/orm/prisma/package.json]: no just for the peerDependencies, to allow future major version.

**Review by Romakita — APPROVED** [review]

**Pull request merged**